src/printer/cvc/cvc_printer.cpp

```diff
@@ -139,7 +139,7 @@ void CvcPrinter::toStream(
   if(n.getMetaKind() == kind::metakind::CONSTANT) {
     switch(n.getKind()) {
     case kind::BITVECTOR_TYPE:
-      out << "BITVECTOR(" << n.getConst<BitVectorSize>().d_size << ")";
+      out << "BITVECTOR(" << n.getConst<BitVectorSize>().d_size << ')';
```

Comment:
Why this change?

Reply:
I have a personal preference for characters. Though, it looks like the change affected code unrelated to my PR. Reverted.